python/meeting-scheduling/src/meeting_scheduling/demo_data.py

```diff
@@ -0,0 +1,214 @@
+            minutes_of_day = current_time.hour * 60 + current_time.minute
+            
+            time_grains.append(TimeGrain(
+                id=str(count),
+                grain_index=count,
+                day_of_year=day_of_year,
+                starting_minute_of_day=minutes_of_day
+            ))
+            
+            count += 1
+            current_time += timedelta(minutes=15)  # 15-minute increments
+        
+        current_date += timedelta(days=1)
+    
+    return time_grains
+
+
+def generate_meetings(people: List[Person], rnd: Random) -> List[Meeting]:
+    """Generate meetings with topics and attendees."""
+    meeting_topics = [
+        "Strategize B2B", "Fast track e-business", "Cross sell virtualization",
+        "Profitize multitasking", "Transform one stop shop", "Engage braindumps",
+        "Downsize data mining", "Ramp up policies", "On board synergies",
+        "Reinvigorate user experience", "Strategize e-business", "Fast track virtualization",
+        "Cross sell multitasking", "Profitize one stop shop", "Transform braindumps",
+        "Engage data mining", "Downsize policies", "Ramp up synergies",
+        "On board user experience", "Reinvigorate B2B", "Strategize virtualization",
+        "Fast track multitasking", "Cross sell one stop shop", "Reinvigorate multitasking"
+    ]
+    
+    meetings = []
+    for i, topic in enumerate(meeting_topics):
+        meeting = Meeting(id=str(i), topic=topic, duration_in_grains=0)
+        meetings.append(meeting)
+    
+    # Set durations using CountDistribution and random.choices
+    duration_distribution = (
+        CountDistribution(count=8, weight=1),   # 33% with 8 time grains
+        CountDistribution(count=12, weight=1),  # 33% with 12 time grains
+        CountDistribution(count=16, weight=1)   # 33% with 16 time grains
+    )
+
+    for meeting in meetings:
+        duration_time_grains, = rnd.choices(population=counts(duration_distribution),
+                                           weights=weights(duration_distribution))
+        meeting.duration_in_grains = duration_time_grains
+    
+    # Add required attendees using CountDistribution and random.choices
+    required_attendees_distribution = (
+        CountDistribution(count=2, weight=0.36),
+        CountDistribution(count=3, weight=0.08),
+        CountDistribution(count=4, weight=0.02),
+        CountDistribution(count=5, weight=0.08),
+        CountDistribution(count=6, weight=0.10),
+        CountDistribution(count=7, weight=0.05),
+        CountDistribution(count=8, weight=0.05),
+        CountDistribution(count=10, weight=0.05)
+    )
+
+    def add_required_attendees(meeting: Meeting, count: int) -> None:
+        while len(meeting.required_attendances) < count:
+            person = rnd.choice(people)
+            try:
+                meeting.add_required_attendant(person)
+            except ValueError:
+                pass
```

Comment:
Can be simplified using [random.sample](https://docs.python.org/3/library/random.html#random.sample):
```suggestion
        meeting.required_attendances = rnd.sample(people, k=count)
```